tests/testthat/_snaps/verb-compute.md

```diff
@@ -21,7 +21,7 @@
       df %>% compute(name = in_schema("main", "db1"), temporary = FALSE)
     Condition
       Error in `db_save_query.DBIConnection()`:
-      ! Can't save query to table `main`.`db1`.
+      ! Can't save query to table main.db1.
```

Comment:
Is this intended?

Reply:
Yes, this is on purpose:

* this might help to disencourage people to use `format()` when they should actually use `escape()`
* the backticks aren't the correct way to escape names in many databases. So, the error message might confuse people that dbplyr didn't escape the table identifier correctly.

Reply:
We do generally prefer to use some indicator of the start and end of the string because it's not uncommon to accidentally add a leading or trailing space, which is then really hard to discover.